developer/src/common/build.sh

```diff
@@ -11,5 +11,8 @@ builder_describe \
   ":delphi                      Delphi components" \
   ":web                         Web components"
 
+builder_describe_platform \
+  :delphi win,delphi
```

Comment:
Is this conflating Delphi components with "platforms"? 
For terminology, I would have thought platform was top level like: android, ios, mac, linux, windows, developer, web

Reply:
I hear you.

I used platform / tool somewhat interchangeably here. I didn't want two functions for this, and it seemed like 'platform' was close enough to cover both operating system and toolchain/toolset/tool. Platform here means the actual machine the build is running on. Tools means the actual compilers available on the machine the build is running on.

I avoided 'component' or 'target' because those have other meanings.